### ISSUE TITLE
Insertion codes not supported

### ISSUE THREAD

**Issue by RasmusFonseca** [getcontacts/getcontacts#50]
A student of Prof Deanes group pointed out that getcontacts don't support insertion codes. For example, in 3MXW it produces:
```
0       vdw     H:ASP:100:O     L:GLN:89:OE1
0       vdw     H:ASP:100:O     L:ASP:91:CB
0       vdw     H:PHE:100:CB    L:THR:34:OG1
0       vdw     H:PHE:100:CG    L:THR:34:OG1
```

**Comment by mchonofsky** [comment]
See pull request #51

**Comment by RasmusFonseca** [comment]
Fixed by #51. Closing now
